### DISCUSSION TITLE
Assign specific workspace to monitor on connect?

### DISCUSSION THREAD

**Discussion by 0xf0xx0** [i3/i3#4987] [Q&A]
Hello! o7

I use a multi-monitor setup from time to time, and ive noticed i3 takes the first empty workspace when assigning monitors. This affects my workflow, as i usually have workspaces  1-5 on my main display, and use the rest on the others. How can i tell i3 to put a specific workspace on a new monitor when attached?

**Comment by Airblader** [comment]
https://i3wm.org/docs/userguide.html#workspace_screen
